src/views/AcademicCheckIn/components/RaceEthnicity/info.json

```diff
@@ -0,0 +1,9 @@
+{
+    "header": "Federal Reporting Status",
```

Comment:
As above, this doesn't seem like the ideal circumstances to store data as a json file.

Reply:
Yeah, I understand this one. This was I think more because we wanted to be consistent, but in hindsight it was probably not necessary.